src/Faker/Provider/Color.php

```diff
@@ -113,4 +113,26 @@ public static function colorName()
     {
         return static::randomElement(static::$allColorNames);
     }
+
+    /**
+     * @example '340,50,20'
+     */
+    public static function hslColor()
+    {
+        return sprintf(
+            '%s,%s,%s',
+            static::numberBetween(0, 360),
+            static::numberBetween(0, 100),
+            static::numberBetween(0, 100)
+        );
+    }
+
+    public static function hslColorAsArray()
```

Comment:
What do you think about adding a `@return` annotation with `array` as type here?